content-scripts/src/modules/login.js

```diff
@@ -1,73 +1,91 @@
+import { fetchSigarraPage } from "./utilities/pageUtils";
 import { getStorage, setStorage } from "./utilities/storage";
 
 const emptyLogin = { auto_login: { verifed: false, user_info: "" } };
 
-var auto_login;
+var auto_login = {};
 
 export const rememberLogin = async (data) => {
     if (!(data?.autoLogin === "on")) return;
 
+    const isAuthenticated = document.querySelector('#se-auth-form') == null;
+
     auto_login = await getStorage("auto_login");
-    console.log(auto_login);
-    if (auto_login === undefined) {
+    if (auto_login === null) {
```

Comment:
auto_login is being `undefined` for me. Change the conditional or add `?? null` next to getStorage

Reply:
I'd just do `if (!auto_login)`, this would handle both cases